### PR TITLE
Better console errors for jest

### PR DESCRIPTION
**Pull request by Droogans**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "snappit-visual-regression",
-  "version": "0.6.4",
+  "version": "0.6.1",
```
Comment (Droogans) [inline package-lock.json]: this is wrong

File: src/screenshot.ts [hunk 2]
```diff
@@ -57,9 +57,9 @@ export class Screenshot {
         element: WebElementPromise,
     ): Promise<Screenshot> {
         await driver.manage().timeouts().setScriptTimeout(5000);
-        await driver.executeScript(fs.readFileSync(require.resolve("dom-to-image")).toString());
         const pngString = await driver.executeAsyncScript(`
```
Comment (jj4th) [inline src/screenshot.ts]: This won't work.  The script runs within the context of the browser.

Comment (Droogans) [reply src/screenshot.ts]: Ok this is for running the domtoimage command in firefox with a remote selenium server.

That's a really, really far off goal (and the results were less than satisfactory), and so doesn't need to be a part of this PR.

Comment (jj4th) [reply src/screenshot.ts]: Right, didn't notice this was a template string.  But this made me realize we're injecting this script on every screenshot.  Preserving this knowledge for future generations... Andrew discovered that FireFox doesn't like it when you don't include the dom-to-image script... and we might end up using this strategy for remote firefox connections.

**Pull request merged**